docker/.env

```
@@ -177,15 +177,15 @@ SMQ_GOOGLE_REDIRECT_URL=
 # Docker image tag
-SMQ_RELEASE_TAG=latest
+SMQ_RELEASE_TAG=v0.18.3
 
 # Proxy
 UV_CUBE_PROXY_LOG_LEVEL=debug
 UV_CUBE_PROXY_HOST=0.0.0.0
 UV_CUBE_PROXY_PORT=8900
 UV_CUBE_PROXY_SERVER_CERT=
 UV_CUBE_PROXY_SERVER_KEY=
-UV_CUBE_AGENT_URL=http://cube-agent:8901
+UV_CUBE_AGENT_URL=https://10.172.192.41:6193
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UV_CUBE_AGENT_URL key should go before the UV_CUBE_PROXY_HOST key

(UnorderedKey)